runtime/doc/api.txt

```diff
@@ -3336,6 +3336,12 @@ nvim_set_decoration_provider({ns_id}, {opts})
                    included. >
                     ["range", winid, bufnr, begin_row, begin_col, end_row, end_col]
 <
+                   In addition to returning a boolean, it is also allowed to
+                   return a `skip_row, skip_col` pair of integers. This
```

Comment:
Any reason not to require this return type, and drop support for boolean?

Reply:
> might be more convenient than `return 2147483647`

Could `math.huge` work? 

Generally we favor simpler types/logic, especially for uncommon use-cases (this is a rather low-level interface).

Not a blocker.